### PR TITLE
COC Update

### PR DESCRIPTION
**Pull request by canyon289**

### CODE REVIEW

File: content/codeofconduct.md [hunk 1]
```diff
@@ -33,10 +33,8 @@ If you believe someone is violating the Code of Conduct, please report this in a
 
 You can make a personal or anonymous report through any of the following channels:
 
-Email: Send a message to any or all Diversity Chairs, who can be reached at oriol.abril.pla@gmail.com and andorra.alexandre@gmail.com. Event organizers will respond promptly.
-Slack: Join the PyMCon Slack and contact the Diversity Chairs, Oriol Abril-Pla or Alex Andorra.
-Online Form: A variant of the NumFOCUS Code of Conduct Report is available for PyMCon here (your name and contact information are not required).
-During PyMCon, we recommend contacting via email or Slack so we can take immediate action if necessary. In addition to contacting directly the Diversity committee, we also encourage you to make a report using our reporting form.
+Email: Send a message to pymconference@gmail.com address
```
Comment (OriolAbril) [inline content/codeofconduct.md:36]: Will this email be dedicated to coc issues? Who will be reading the messages that get there? 

Some comments:
* only the code of conduct commitee should have access to coc reports, so if this email is used for other things I would not use it for that
* Already having the report form, if the response time of the email won't be significantly faster than the form then I am not sure it is worth it.

File: content/codeofconduct.md [hunk 2]
```diff
@@ -33,10 +33,8 @@ If you believe someone is violating the Code of Conduct, please report this in a
...
+Online Form: The NumFocus Code of Conduct reporting form is available. https://numfocus.org/code-of-conduct
```
Comment (OriolAbril) [inline content/codeofconduct.md:37]: I would have that link be directly to the report form.

File: content/codeofconduct.md [hunk 3]
```diff
@@ -73,7 +71,7 @@ All reports of breaches of the code of conduct will be investigated and handled
 
 ## The current PyMCon Diversity Committee consists of:
```
Comment (OriolAbril) [inline content/codeofconduct.md:72]: All the "persons responsible for resolving complaints" is gone, and only this is left, formatted as a heading. If we only use numfocus form, then we don't need this section, _nor the conflict of interest one_ as they are not pymcon specific.

Similar comment applies to the timeline summary though. It most probably updated to reflect what will really happen. Now it doesn't reallly make any sense, there is no diversity committee, I am also not sure how much info will be available to us for example, will it be possible to make the summary of reported incidents?

**Review by OriolAbril — DISMISSED** [review]
If you compare https://pymcon.com/codeofconduct/ vs https://pymc-devs.github.io/pymcon//code_of_conduct there are still many things missing.

One of the first things that pop to your eyes when getting to the coc page should be a link to the report form (orange button in 2020 page). This is completely missing from 2022 page. Towards the bottom there is also the "read the numfocus code of conduct" which linked to the full code of conduct and this link is also gone.

The how to report section is very hard to read in 2022 because the text gets wrapped, they are not bullet points. This also happens in all other bullet points from 2020, none has been preserved.

**Comment by canyon289** [comment]
@OriolAbril would you be able to make the updates, otherwise Ill get back to this in a couple of weeks

**Comment by OriolAbril** [comment]
I'll try to make some of the changes, not sure I'll be able to beat you to it though :/

**Comment by canyon289** [comment]
No problem, right now were not at any risk of Code of Conduct violation so there's time. As long as we get this done before the first set of events we should be good.

**Comment by OriolAbril** [comment]
also cc @canyon289 because I can't ask you to review (I assume because you are the author of the PR)

File: data/faqs.toml [hunk 4]
```diff
@@ -12,3 +12,7 @@ description = "Can't find the answer you're looking for? We've shared some of ou
 [[faq]]
     question = "What is the branding guide for PyMCon?"
     answer = "The branding guide can be found 👉 <a href='https://github.com/pymc-devs/brand' target='_blank'> here </a>"
+
+[[faq]]
```
Comment (OriolAbril) [inline data/faqs.toml:16]: not very convinced on the actual question and answer but I do want to have one faq about the code of conduct encouraging people to report and against the "code of conduct violations are super grave/lifetime expulsion grave" pre-conception.

Comment (canyon289) [reply data/faqs.toml:16]: Thats fine adding the section in as a start is great

**Review by twiecki — APPROVED** [review]

File: content/codeofconduct.md [hunk 5]
```diff
@@ -25,86 +31,84 @@ Thank you for helping make this a welcoming, friendly community for all.
...
-## Appealing a Decision
+Reports sent via the additional event specific channels will be handled by members of the
+PyMCon Code of Conduct Enforcement Team, consulting the NumFOCUS team if necessary.
 
-To appeal a decision, contact Oriol Abril-Pla at oriol.abril.pla@gmail.com with your appeal and the Executive committee or the NumFOCUS Code of Conduct Enforcement Team will review the case.
+### PyMCon Code of Conduct Enforcement Team
+Members of the PyMCon Code of Code Enforcement team are:
 
-## Timeline Summary:
+* Oriol Abril Pla `oriol.abril.pla@gmail.com`
```
Comment (canyon289) [inline content/codeofconduct.md:92]:
```suggestion
* Oriol Abril Pla `oriol.abril.pla@gmail.com`
* Ravin Kumar `rkumar@pymc_devs.com
```

**Comment by canyon289** [comment]
I cant approve my own PR, so consider this approval of changes. Merging

**Pull request merged**